deeplay/components/transformer/satt.py

```diff
@@ -0,0 +1,61 @@
+from deeplay import DeeplayModule, Layer
+
+import torch
+import torch.nn as nn
+
+
+class MultiheadSelfAttention(DeeplayModule):
```

Comment:
Should also contain some form of documentation, e.g. what is now in the forward-pass

Reply:
updated